debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/IncrementalSnapshotIT.java

```diff
@@ -125,6 +126,11 @@ protected String returnedIdentifierName(String queriedID) {
         return queriedID.toUpperCase();
     }
 
+    @Override
+    protected Optional<String> physicalRowIdentifierSurrogateKey() {
+        return Optional.of("\"ROWID\"");
```

Comment:
Not that I think quoting this will be problematic as Oracle stores this in uppercase, but I am more curious what is the reason for needing the quotes?